tests/netuit/assign/AssignSegregated.cpp

```diff
@@ -0,0 +1,13 @@
+#include <mpi.h>
+
+#define CATCH_CONFIG_MAIN
+#define CATCH_CONFIG_DEFAULT_REPORTER "multiprocess"
+#include "Catch/single_include/catch2/catch.hpp"
+
+#include "netuit/assign/AssignSegregated.hpp"
+#include "uitsl/debug/MultiprocessReporter.hpp"
+#include "uitsl/mpi/MpiGuard.hpp"
+
+const uitsl::MpiGuard guard;
+
+// stub test
```

Comment:
can you add a TEST_CASE() {  that just instantiates an instance of the object? and mark them TODO stub test for these